### DISCUSSION TITLE
Reset and migrate database without seeding

### DISCUSSION THREAD

**Discussion by robertcoopercode** [prisma/prisma#10024] [Q&A]
Is it possible to run `prisma migrate reset` without have the seeds applied?

**Comment by ruheni** [comment]
Hi @robertcoopercode 👋🏽

Alex from the Prisma team here. 

Yes, this should be possible by passing `--skip-seed` option when running `prisma migrate reset`.

You can check out further details on the available options in our [docs](https://www.prisma.io/docs/reference/api-reference/command-reference#migrate-reset)

**Reply by heymartinadams** [reply]
Looks like `--skip-seed` was removed with Prisma 7. https://www.prisma.io/docs/orm/more/upgrade-guides/upgrading-versions/upgrading-to-prisma-7#cli-flags---skip-generate-and---skip-seed-removed

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
